src/Components/Components/test/EventCallbackTest.cs

```diff
@@ -0,0 +1,457 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Components
+{
+    public class EventCallbackTest
+    {
+        [Fact]
+        public async Task EventCallback_Default()
+        {
+            // Arrange
+            var callback = default(EventCallback);
+
+            // Act & Assert (Does not throw)
+            await callback.InvokeAsync(null);
+        }
+
+        [Fact]
+        public async Task EventCallbackOfT_Default()
+        {
+            // Arrange
+            var callback = default(EventCallback<UIEventArgs>);
+
+            // Act & Assert (Does not throw)
+            await callback.InvokeAsync(null);
+        }
+
+
```

Comment:
Nit: unnecessary blank line here, and in a few other places in this file. I don't really mind whether it gets fixed. I'm kind of only posting this as proof that I'm actually paying attention.

Reply:
Cools beans. I might leave this here, Pranav will be overjoyed if he gets the chance to remove it later.